src/EfficiencyHists.cxx

```diff
@@ -0,0 +1,34 @@
+#include "TrackPerf/EfficiencyHists.hxx"
+
+#include <EVENT/Track.h>
+#include <EVENT/MCParticle.h>
+
+using namespace TrackPerf;
+
+EfficiencyHists::EfficiencyHists(bool effi) {
+  if (effi) {
+     h_effpt = new TEfficiency("pt_vs_eff", ";Truth pT [GeV]; Efficiency", 50, 0, 12 );
+     h_efftheta = new TEfficiency("theta_vs_eff", ";Truth theta; Efficiency", 50, -4, 4);
+  } else {
+    h_effpt = new TEfficiency("fake_pt_vs_eff",
+      "Fake Track Rate vs pT;Reconstructed pT [GeV]; Efficiency", 50, 0, 12);
```

Comment:
I think you also want to change the y-axis label from "Efficiency" to "Fake Rate".